Plaid API Endpoints Collection.json

```diff
@@ -6388,6 +6489,203 @@
 							"body": "{\n \"request_id\": \"mdqfuVxeoza6mhu\"\n}"
 						}
 					]
+				},
+				{
+					"name": "Get a sweep",
```

Comment:
I was unable to test these in postman (even in Sandbox) but they look okay in the Postman app.